pkg/tsdb/influxdb/response_parser_bench_test.go

```diff
@@ -0,0 +1,33 @@
+package influxdb
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+// go test -benchmem -run=^$ -memprofile memprofile.out -count=10 -bench ^BenchmarkParseJson$ github.com/grafana/grafana/pkg/tsdb/influxdb
+// go tool pprof -http=localhost:9999 memprofile.out
+func BenchmarkParseJson(b *testing.B) {
+	responseFileName := filepath.Join("./testdata", "response.json")
+
+	// nolint:gosec
+	// We can ignore the gosec G304 warning since this is a test file
+	responseBytes, err := os.ReadFile(responseFileName)
```

Comment:
Can you also change the test to not read the file? (I sent you a patch in Slack for that 😅)

Reply:
I've missed that patch. Now it is being embedded https://github.com/grafana/grafana/pull/64776/commits/a921188bdb2283be024849e261eae46bebd11164